docs/how-to/code/internal-user/rockcraft.yaml

```diff
@@ -1,32 +1,36 @@
 name: internal-user
-base: bare
-build-base: ubuntu@24.04
+base: ubuntu@22.04
```

Comment:
Why has this changed?

Reply:
to accomodate for using the npm plugin (see next conversation for details).